router.go

```diff
@@ -91,8 +91,11 @@ func (r *Router) Add(method, path string, h HandlerFunc) {
 	pnames := []string{} // Param names
 	ppath := path        // Pristine path
 
-	if h == nil && r.echo.Logger != nil {
+	if h == nil {
 		// FIXME: in future we should return error
+		if r.echo.Logger == nil {
+			panic("Adding route without handler function and Logger is not init(ed)")
```

Comment:
Please correct to `Adding route without handler function is not allowed with uninitialized logger`